src/Files.App/Views/Properties/GeneralPage.xaml

```diff
@@ -422,17 +442,19 @@
 						FontWeight="SemiBold"
 						Text="{helpers:ResourceString Name=PropertiesDriveFreeSpace/Text}" />
 				</StackPanel>
+
 				<TextBlock
 					x:Name="driveFreeSpaceValue"
 					Grid.Row="1"
 					Grid.Column="2"
 					x:Load="{x:Bind ViewModel.DriveFreeSpaceVisibility, Mode=OneWay}"
 					Style="{StaticResource PropertyValueTextBlock}"
 					Text="{x:Bind ViewModel.DriveFreeSpace, Mode=OneWay}" />
+
 				<Button
 					Grid.Row="1"
 					Grid.Column="3"
-					Padding="8,6"
+					Padding="6,6"
```

Comment:
Can you share a screenshot of the before and after?

Reply:
As above